content/en/docs/devops-user-guide/examples/multi-cluster-project-example.md

```diff
@@ -10,8 +10,8 @@ weight: 11420
 
 - You need to [enable the multi-cluster feature](../../../../docs/multicluster-management/).
 - You need to have a [Docker Hub](https://hub.docker.com/) account.
-- You need to [enable the KubeSphere DevOps System](../../../../docs/pluggable-components/devops/) on your host cluster.
-- You need to create a workspace with multiple clusters, a DevOps project on your **host** cluster, a multi-cluster project (in this tutorial, this multi-cluster project is created on the host cluster and one member cluster), and an account (`project-regular`). This account needs to be invited to the DevOps project and the multi-cluster project with the role `operator`. For more information, see [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/), [Multi-cluster Management](../../../multicluster-management/) and [Multi-cluster Projects](../../../project-administration/project-and-multicluster-project/#multi-cluster-projects).
+- You need to [enable the KubeSphere DevOps System](../../../../docs/pluggable-components/devops/) on your Host Cluster.
+- You need to create a workspace with multiple clusters and use an account (for example, `project-admin`) with the role of `workspace-self-provisioner` to create a multi-cluster project (in this tutorial, this multi-cluster project is created on the Host Cluster and one Member Cluster) and a DevOps project on the Host Cluster. Invite an account (for example, `project-regular`) to the DevOps project and grant it the role of `operator`. For more information, see [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/), [Multi-cluster Management](../../../multicluster-management/) and [Multi-cluster Projects](../../../project-administration/project-and-multicluster-project/#multi-cluster-projects).
```

Comment:
What about using one account? I think an account with `project-admin` role is enough for both devops project and multi-cluster project.

Reply:
It would be better if we can use different accounts with different roles.